### DISCUSSION TITLE
Feature Request: RealtimeDisconnectEvent hook

### DISCUSSION THREAD

**Discussion by benallfree** [pocketbase/pocketbase#467] [Q&A]
Looking at https://github.com/pocketbase/pocketbase/blob/master/apis/realtime.go#L98, it seems like that would be a perfect place to add a `RealtimeDisconnectEvent`

I need this for my application because I need to implement user presence and clean up when the user disconnects. 

If this is not already in the works, I'd be happy to submit a PR for it.

**Comment by ganigeorgiev** [comment]
This could be added. Event hooks for the broadcasted message also would be nice to have to further validate whether the client can receive the event (this was discussed partially in https://github.com/pocketbase/pocketbase/discussions/395#discussioncomment-3585686).

I'll create shortly an issue so that it can be tracked in the roadmap, but this will be handled after the ongoing users and profiles refactoring in #376.

**Reply by ganigeorgiev** [reply]
_Here is the issue - https://github.com/pocketbase/pocketbase/issues/468._
